docs/en/enterprise-edition/content-collections/dashboards/dashboards-vulnerabilities.adoc

```diff
@@ -39,7 +39,7 @@ image::dashboards/uve-dashboard-c2c.gif[]
 
 [NOTE]
 ====
-The dashboard is only available to users with the System Administrator role on all stacks, with the exception of http://app.gov[app.gov] and http://app.cn[app.cn.]
+The dashboard is only available to users with the System Administrator role on all stacks, with the exception of http://app.gov[app.gov], http://app.cn[app.cn.]. http://app.gov[app.gov] users with a business license also do not have access to the dashboard.
```

Comment:
@jenjoe22 Suggesting a rephrase as follows:
The dashboard is only available to users with the System Administrator role on all stacks, except for http://app.gov[app.gov] and http://app.cn[app.cn] stacks.  Additionally, http://app.gov[app.gov] stack users with a business license will not have access to the dashboard.